modules/debugsp/templates/status.twig

```diff
@@ -0,0 +1,100 @@
+{% set pagetitle = 'SimpleSAMLphp installation page'|trans %}
```

Comment:
This seems like an exact copy.. Can't we just use the `auth_status.twig` from the base templates-dir?

Reply:
Much cleaner using the base template. I had to define a trackid to help the base template. This came from the status.twig in the admin module so it is also very similar to the base `auth_status.twig`.

Reply:
This should be resolved in https://github.com/simplesamlphp/simplesamlphp/pull/2381/commits/8aeee60314b0862d6bdfc707793b9240d666101a